### PR TITLE
Removes Xenoarch access on Mining Shuttle

### PR DESCRIPTION
**Pull request by Novacat**
On request from Admins

### CODE REVIEW

**Comment by Novacat** [comment]
I am not too worried about the doors leading into the area. Just the shuttle itself.

**Comment by TheFurryFeline** [comment]
If xenoarch is to lose access as a whole, the doors should also have xenoarch access removed.

**Comment by KasparoVy** [comment]
Cheers.

File: code/modules/shuttles/shuttles_vr.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 /obj/machinery/computer/shuttle_control/belter
 	name = "belter control console"
-	req_one_access = list(access_mining, access_xenoarch, access_medical_equip) //Allows xenoarch, miners AND doctors to use it.
+	req_one_access = list(access_mining, access_medical_equip) //Allows xenoarch, miners AND doctors to use it.
```
Comment (KasparoVy) [inline code/modules/shuttles/shuttles_vr.dm:13]: Comment wasn't changed, eh?

Comment (TheFurryFeline) [reply code/modules/shuttles/shuttles_vr.dm:13]: Good catch.

**Comment by Heroman3003** [comment]
If Xenoarch are not expected to do belt mining anymore, belt miner crate might need a change, because I don't remember exactly, but it also might have xenoarch access.

**Pull request merged**